tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/Main.java

```diff
@@ -0,0 +1,152 @@
+            .forHelp();
+        final OptionSet options;
+        try {
+            options = parser.parse(args);
+        } catch (final OptionException ex) {
+            parser.printHelpOn(System.out);
+            throw ex;
+        }
+        final LsVersionType type;
+        final String version;
+        if (options.has(distribution)) {
+            type = LsVersionType.DISTRIBUTION;
+            version = options.valueOf(distribution);
+        } else if (options.has(git)) {
+            type = LsVersionType.GIT;
+            version = options.valueOf(git);
+        } else {
+            type = LsVersionType.LOCAL;
+            version = options.valueOf(local);
+        }
+        execute(
+            new UserOutput(System.out), loadSettings(), options.valueOf(testcase),
+            options.valueOf(pwd).toPath(), version, type
+        );
+    }
+
+    public static void execute(final UserOutput output, final Properties settings,
+        final String test, final Path cwd, final String version, final LsVersionType type)
+        throws Exception {
+        output.printBanner();
+        output.printLine();
+        output.green(String.format("Benchmarking Version: %s", version));
+        output.green(String.format("Running Test Case: %s", test));
+        output.printLine();
+        Files.createDirectories(cwd);
+        final LogstashInstallation logstash = LsBenchLsSetup.setupLS(
+            cwd.toAbsolutePath().toString(), version, type,
+            JRubyInstallation.bootstrapJruby(cwd, JRubyInstallation.JRUBY_DEFAULT_VERSION)
+        );
+        final Case testcase;
+        if ("baseline".equals(test)) {
+            testcase = new GeneratorToStdout(logstash);
+        } else if ("apache".equals(test)) {
+            testcase = new ApacheLogsComplex(logstash, cwd, settings);
+        } else {
+            throw new IllegalArgumentException(String.format("Unknown test case %s", test));
+        }
+        output.printStartTime();
+        output.green("Statistical Summary:\n");
+        final long start = System.currentTimeMillis();
+        final EnumMap<LsMetricStats, ListStatistics> stats = testcase.run();
+        output.green(String.format(
+            "Elapsed Time: %ds",
+            TimeUnit.SECONDS.convert(
+                System.currentTimeMillis() - start, TimeUnit.MILLISECONDS
+            )
+        ));
+        output.green(String.format("Num Events: %f", stats.get(LsMetricStats.COUNT).getMax()));
+        final ListStatistics throughput = stats.get(LsMetricStats.THROUGHPUT);
+        output.green(String.format("Min Throughput: %f", throughput.getMin()));
+        output.green(String.format("Max Throughput: %f", throughput.getMax()));
+        output.green(String.format("Mean Throughput: %f", throughput.getMean()));
+        output.green(String.format("StdDev(Throughput): %f", throughput.getStandardDeviation()));
+        output.green(String.format("Variance(Throughput): %f", throughput.getVariance()));
```

Comment:
more nitpicking: Switch from ```X Throughput``` to ```Throughput X``` and remove the parentheses for StdDev and Variance.